users/views.py

```diff
@@ -202,3 +208,17 @@ def update_organization_user(sender, instance: OrganizationUser, **kwargs):
     async_to_sync(channel_layer.group_send)(
         user_group_name, {"type": "send_user", "data": user_data}
     )
+
+
+@api_view(["POST"])
+def retrieve_analytics_app_access_token(request):
+    """ Makes a client credentials request to Auth0 app to get an access token. """
+    url = AUTH0_TOKEN_URL
+    payload = {
+        "grant_type": "client_credentials",
+        "client_id": AUTH0_CLIENT_ID,
+        "client_secret": AUTH0_CLIENT_SECRET,
+        "audience": AUTH0_AUDIENCE,
+    }
+    response = requests.post(url, data=payload)
```

Comment:
any reason to not do?
```suggestion
    payload = {
        "grant_type": "client_credentials",
        "client_id": AUTH0_CLIENT_ID,
        "client_secret": AUTH0_CLIENT_SECRET,
        "audience": AUTH0_AUDIENCE,
    }
    response = requests.post(AUTH0_TOKEN_URL, data=payload)
```

Reply:
just a miss. I had it copy-pasted